
<!DOCTYPE html>
<html lang="en">
<head>
<meta charset="UTF-8">
<link rel="stylesheet" type="text/css" href="css/GENEDBQuery.css"/>
<link rel="stylesheet" type="text/css" href="css/IndividualResult.css"/>
<link type="text/css" href="css/GENEDBQuery_old.css" rel="stylesheet" />
<link type="text/css" href="css/styleFoot.css" rel="stylesheet" />
<title>IMGT/GENE-DB Consultation</TITLE>
</head>
<body id="GENE-DB">
<table style="width:100%;">
<tr>
<td align="left" nowrap valign="middle" width="70%">
<font size="+3" color="#000099"><b>THANK YOU</b><br/>
<b>for using <a href="/genedb/doc">IMGT/GENE-DB</a></b></font>
</td>
<td class="notshow" align=right valign=top width="30%"><a href="/"><img SRC="images/logo_IMGT2.png" ALT="WELCOME !" class="mainlogo"/></a>
</td>
</tr>
<tr>
<td class="logolegendleft">IMGT&reg;, the international ImMunoGeneTics information system&reg;</td>
<td class="logolegend">http://www.imgt.org</td>
</tr>
</table>
<hr/>
<img src="images/logoGene-DB.png" ALT="Gene-DB logo" class="mainlogo" />

<table class="citation">
<tr>
<td class="citation">
<b>Citing IMGT/GENE-DB</b>: Giudicelli, V. et al. Nucleic Acids Res., 33: D256 - D261 (2005).
PMID: <a href="https://pubmed.ncbi.nlm.nih.gov/15608191/">15608191</a>
<a href="/PDF/NAR/33_D256-D261_2005.pdf"><img src="/images/pdficon.png" alt="pdf" /></a>
</td>
</tr>
</table>
<p class="programversion">IMGT/GENE-DB program version: <a href="/IMGTgenedbdoc/programversions.html">3.1.31</a> (16 December 2020)<br/>
<a href="/IMGTgenedbdoc/dataupdates.html">IMGT/GENE-DB data updates </a>
</p>

<h2 class="subtitle"><a href="/IMGTScientificChart/SequenceDescription/IMGTreferencesequences.html" >IMGT/GENE-DB reference sequences</a> in FASTA format:</h2>
<h2 class="subtitle">Amino acids  sequences with gaps according to the <a href="/IMGTScientificChart/Numbering/IMGTnumberingCR.html">IMGT unique numbering</a><br /> for <span class="casimir">F+ORF+in-frame P with IMGT Gaps</span> alleles including orphons</h2>
<hr>
<pre>
The FASTA header contains 15 fields separated by '|':

1. IMGT/LIGM-DB accession number(s)
2. IMGT gene and allele name
3. species
4. IMGT allele functionality
5. exon(s), region name(s), or extracted label(s)
6. start and end positions in the IMGT/LIGM-DB accession number(s)
7. number of nucleotides in the IMGT/LIGM-DB accession number(s)
8. codon start, or 'NR' (not relevant) for non coding labels
9. +n: number of nucleotides (nt) added in 5' compared to the corresponding label extracted from IMGT/LIGM-DB
10. +n or -n: number of nucleotides (nt) added or removed in 3' compared to the corresponding label extracted from IMGT/LIGM-DB
11. +n, -n, and/or nS: number of added, deleted, and/or substituted nucleotides to correct sequencing errors, or 'not corrected' if non corrected sequencing errors
12. number of amino acids (AA): this field indicates that the sequence is in amino acids
13. number of characters in the sequence: nt (or AA)+IMGT gaps=total
14. partial (if it is)
15. reverse complementary (if it is)
</pre>
<hr/>
<b>Number of results = 170</b><br />
<pre>
>IMGT000063|IGKV1-101*01|Macaca mulatta_AG07107|F|V-REGION|46222..46508|287 nt|1| | | |95 AA|95+17=112| |rev-compl|
AIQMTQSPSSLSVSVGDGVT.ITCQASQGI......SNELAWYQQKPGNSPKLLIYKA..
.....SNLQTGLA.SQFIGGG..SGIDFTLTINNIQPDEVATYYCQQENSSP
>IMGT000063|IGKV1-16*01|Macaca mulatta_AG07107|F|V-REGION|1128341..1128627|287 nt|1| | | |95 AA|95+17=112| | |
DIQMTQSPSSLSASVGDKVT.ITCQASQSI......SSWLAWYQQKPGKAPKPLIYKA..
.....SSLESGVP.SRFSGSG..SGTDFTLTISSLQPEDFATYYCQQYNSAP
>IMGT000063|IGKV1-18*01|Macaca mulatta_AG07107|F|V-REGION|1111209..1111495|287 nt|1| | | |95 AA|95+17=112| | |
DIQMTQSPSSLSASVGDKVT.ITCRASQGI......SSWLAWYQQKPGKAPKLLIYAA..
.....SSLQSGVP.SRFSGSG..SGTDYTLTISSLQPEDFATYYCQQGYNTP
>IMGT000063|IGKV1-19*01|Macaca mulatta_AG07107|F|V-REGION|1092181..1092467|287 nt|1| | | |95 AA|95+17=112| | |
DIQMTQSPSSLSASVGDKVT.ITCHASQGI......SSWLAWYQQKPGKAPKPLIYAA..
.....SSLQSGVP.SRFSGSG..SGTDYTLTISSLQPEDFATYYCQQYDDLP
>IMGT000063|IGKV1-21*01|Macaca mulatta_AG07107|F|V-REGION|1053655..1053941|287 nt|1| | | |95 AA|95+17=112| | |
DIQMTQSPSSLSASVGDRVT.ITCRASQGI......SSWLAWYQQKPGKAPKLLIYKA..
.....SSLQSGVP.SRFSGSG..SGTDFTLTISSLQPEDFATYYCQQYNSAP
>IMGT000063|IGKV1-22*01|Macaca mulatta_AG07107|F|V-REGION|1035576..1035862|287 nt|1| | | |95 AA|95+17=112| | |
DIQMTQSPSSLSASVGDTVT.ITCRASQSI......SSWLAWYQQKPGKAPKLLIYKA..
.....SSLQSGVP.SRFSGSG..SGTDFTLTISSLQPEDFATYYCLQYSSSP
>IMGT000063|IGKV1-25*01|Macaca mulatta_AG07107|F|V-REGION|998503..998789|287 nt|1| | | |95 AA|95+17=112| | |
DIQMTQSPSSLSASVGDRVT.ITCRASQGI......SSYLAWYQQKPGKAPKLLIYKA..
.....STLQSGVP.SRFSGSG..SGTDFTLTISSLQPEDFATYYCQQHNSNP
>NW_001099007|IGKV1-25*02|Macaca mulatta_17573|F|V-REGION|368713..368999|287 nt|1| | | |95 AA|95+17=112| |rev-compl|
DIQMTQSPSSLSASVGDRVT.ITCRASQGI......SSYLAWYQQKPGKAPKLLIYAA..
.....SSLQSGVP.SRFSGSG..SGTDFTLTISSLQPEDFATYYCQQYNSNP
>NW_001099007|IGKV1-25-1*01|Macaca mulatta_17573|P|V-REGION|388350..388636|287 nt|1| | | |95 AA|95+17=112| |rev-compl|
DIQMTQSPCFLSASVGDTVA.ITCQASGGI......SNNLAWYQQKPGKSPKLFLYDA..
.....KDLCTGVS.SKFSGRG..SGTDFTLTIRSLELEDFVAYYCEQKFSYP
>IMGT000063|IGKV1-27*01|Macaca mulatta_AG07107|F|V-REGION|963567..963853|287 nt|1| | | |95 AA|95+17=112| | |
DIQMTQSPSSLSASVGDRVT.ITCQASQGI......SSWLAWYQQKPGKAPKLLLYKA..
.....PGLQSGVP.SMFSGSG..SGTDFTLTISSLQPEYFATYYCQQFSSAL
>IMGT000063|IGKV1-28*01|Macaca mulatta_AG07107|F|V-REGION|956726..957012|287 nt|1| | | |95 AA|95+17=112| | |
DIQMTQSPSSLSASVGDTVT.ITCRASQGI......SSYLNWFQQKPGKAPKLLIYAA..
.....TTLQSGVP.SRFSGSG..SGTEFTLTISSLQPEDFAAYYCLQHNSYP
>NW_001099007|IGKV1-28*02|Macaca mulatta_17573|F|V-REGION|434329..434615|287 nt|1| | | |95 AA|95+17=112| |rev-compl|
DIQMTQSPSSLSASVGDTVT.ITCRASQGI......SSYLNWFQQKPGKAPKLLIYAA..
.....SSLESGVP.SRFSGSG..SGTEFTLTISSLQPEDFATYYCLQYKSYP
>AY963715|IGKV1-28*03|Macaca mulatta|F|V-REGION|159..445|287 nt|1| | | |95 AA|95+17=112| | |
DIQMTQSPSSLSASVGDTVT.ITCRASQGI......SSYLNWFQQKPGKAPKLLIYAA..
.....TTLQSGVP.SRFSGSG..SGTDFTLTISSLQPEDFAAYYCLQHNSYP
>IMGT000063|IGKV1-29*01|Macaca mulatta_AG07107|P|V-REGION|937494..937780|287 nt|1| | | |95 AA|95+17=112| | |
DIKMT*SPFFLSASVGNRVT.ITFQATEGI......SNDLPWYQQKPGKSPKLFLYDA..
.....KDLHTGVS.SRFGGSG..SGTDFTHTISSLDPEDFATYYSKQEFSHS
>NW_001099007|IGKV1-29*02|Macaca mulatta_17573|P|V-REGION|453532..453818|287 nt|1| | | |95 AA|95+17=112| |rev-compl|
DIKMT*SPFFLSASVGNRVT.ITFQATEGI......SNDLPWYQQKPGKSPKLFLYDA..
.....KDLHTGVS.SRFGGSG..SGTDFTHTISSLEPEDFATYYSKQEFSHS
>IMGT000063|IGKV1-32*01|Macaca mulatta_AG07107|F|V-REGION|881793..882079|287 nt|1| | | |95 AA|95+17=112| | |
DIQMTQSPSSLSASVGDRVT.ITCRASQGI......SSYLNWYQQKPGKAPKLLIYYA..
.....NRLESGVP.SRFSGSG..SGTEFTLTISSLQPEDFATYYCQQYNSLP
>NW_001099007|IGKV1-32*02|Macaca mulatta_17573|F|V-REGION|502996..503282|287 nt|1| | | |95 AA|95+17=112| |rev-compl|
DIQMSQSPSSLSASVGDRVT.ITCRASQGI......SSYLNWYQQKPGKAPKLLIYYA..
.....NSLASRVP.SRFSGSG..SGTEFTLTISSLQPEDFATYYCQQGNSNP
>AY963732|IGKV1-32*03|Macaca mulatta|F|V-REGION|167..453|287 nt|1| | | |95 AA|95+17=112| | |
DIQMSQSPSSLSASVGDRVT.ITCRASQGI......SSYLNWYQQKPGKAPKLLIYYA..
.....NSLASGVP.SRFSGSG..SGTEFTLTISSLQPEDFAAYYCLQGYSTP
>AY963734|IGKV1-32*04|Macaca mulatta|F|V-REGION|167..453|287 nt|1| | | |95 AA|95+17=112| | |
DIQMSQSPSSLSASVGDRVT.ITCRASQGI......SSYLNWYQQKPGKAPKLLIYYA..
.....NSLASGVP.SRFSGSG..SGTEFTLTISSLQPEDFATYYCQQGNSNP
>AY963737|IGKV1-32*05|Macaca mulatta|F|V-REGION|165..451|287 nt|1| | | |95 AA|95+17=112| | |
DIQMTQSPSSLSASVGDRVT.ITCRASQGI......SSYLNWYQQKPGKAPKLLIYYA..
.....NRLESGVP.SRFSGSG..SGTEFTLTISSLQPEDFATYYCQQYNSLP
>IMGT000063|IGKV1-33*01|Macaca mulatta_AG07107|F|V-REGION|865581..865867|287 nt|1| | | |95 AA|95+17=112| | |
DIQMTQSPSSLSASVGDKVT.ITCRASQGI......SNALAWYQQKPGKAPKLLIYAA..
.....SNLQSGVP.SRFSGSG..SGTDFTLTISSLQPEDFAVYYCQQRNSYP
>AY963730|IGKV1-33*02|Macaca mulatta|F|V-REGION|167..453|287 nt|1| | | |95 AA|95+17=112| | |
DIQMTQSPSSLSASVGDKVT.ITCRASQGI......SNALAWYQQKPGKAPKLLIYAA..
.....SSLQSGVP.SRFSGSG..SGTDFTLTISSLQPEDFATYYCQQHNSNP
>IMGT000063|IGKV1-36*01|Macaca mulatta_AG07107|F|V-REGION|805792..806078|287 nt|1| | | |95 AA|95+17=112| | |
DIQMTQSPSSLSASVGDRVT.ITCRASQGI......SNYLSWYQQKPGKAPKLLIYAA..
.....SSLESGVP.SRFSGSG..SGTEFTLTISSLQPEDFAAYYCLQYNSKP
>NW_001099007|IGKV1-36*02|Macaca mulatta_17573|F|V-REGION|583943..584229|287 nt|1| | | |95 AA|95+17=112| |rev-compl|
DIQMTQSPSSLSASVGDRVT.ITCRASQGI......SDYLSWYQQKPGKAPKRLIYAA..
.....SSLESGVP.SRFSGSG..SGTEFTLTISSLQPEDFAAYYCLQGYSTP
>AY963731|IGKV1-36*03|Macaca mulatta|F|V-REGION|168..454|287 nt|1| | | |95 AA|95+17=112| | |
DIQMTQSPSSLSASVGDRVT.ITCRASQGI......SDYLSWYQQKPGKAPKLLIYAA..
.....SSLESGVP.SRFSGSG..SGTEFTLTISSLQPEDFAAYYCLQGYSTP
>IMGT000063|IGKV1-37*01|Macaca mulatta_AG07107|F|V-REGION|788811..789097|287 nt|1| | | |95 AA|95+17=112| | |
DIQMTQSPSSLSASVGDRVT.ITCRASQGI......SSYLAWYQQKPGKAPKPLIYYA..
.....SNLESGVP.SRFSGSG..SGTEFTLTISSLQPEDFAIYYCQQYNSDP
>NW_001099007|IGKV1-37*02|Macaca mulatta_17573|P|V-REGION|600847..601133|287 nt|1| | | |95 AA|95+17=112| |rev-compl|
DIQMTQSPSSLSASVGDRVT.ITCRASQGI......SSYLAWYQQKPGKAPKPLIYYA..
.....SNLESGVP.SRFSGSG..SGTEFTLTISSLQPEDFATYYCQQYNSDP
>IMGT000063|IGKV1-38*01|Macaca mulatta_AG07107|F|V-REGION|782183..782469|287 nt|1| | | |95 AA|95+17=112| | |
DIQLTQSPSSLSASVGDRVT.ITCRASQGI......SSYLAWYQQKPGKAPKLLIYDA..
.....SNLQSGVP.SRFSGSG..SGTDFTLTISSLQPEDFAVYYCQQRNSYP
>NW_001099007|IGKV1-38*02|Macaca mulatta_17573|F|V-REGION|607470..607756|287 nt|1| | | |95 AA|95+17=112| |rev-compl|
DIQLTQSPSSLSASVGDRVT.ITCRASQGI......SSYLAWYQQKPGKAPKLLIYKA..
.....SSLQSGVP.SRFSGSG..SGTEFTLTISSLQPEDFAVYYCQQRNSYP
>IMGT000063|IGKV1-41*01|Macaca mulatta_AG07107|F|V-REGION|724925..725211|287 nt|1| | | |95 AA|95+17=112| | |
DIQMTQSPSSLSASVGDRVT.ITCRASQGI......SNYLNWYQQEPGKAPKLLIYAA..
.....SSLESGVP.SRFSGSG..SGTEFTLTISSLQPEDFATYYCQPFKNYL
>NW_001099007|IGKV1-41*02|Macaca mulatta_17573|F|V-REGION|664713..664999|287 nt|1| | | |95 AA|95+17=112| |rev-compl|
DIQMTQSPSSLSASVGDRVT.ITCRASQGI......SNDLNWYQQKPGKAPKLLIYAA..
.....SSLESEVP.SRFSGSG..SGTDFTLTISSLQPEDFATYYCQPFKNYL
>IMGT000063|IGKV1-43*01|Macaca mulatta_AG07107|F|V-REGION|697405..697691|287 nt|1| | | |95 AA|95+17=112| | |
DIQMTQSPSSLSASAGDRVT.ITCRASQGI......STYLNWYQQKPGKAPKRLIYAA..
.....SSLESGVP.SRFSGSG..SGTDFTLTISSLQPEDFATYYCLQYNSNP
>NW_001099007|IGKV1-43*02|Macaca mulatta_17573|F|V-REGION|700108..700394|287 nt|1| | | |95 AA|95+17=112| |rev-compl|
DIQMTQSPSSLSASVGDRVT.ITCRASQGI......STYLNWYQQKPGKAPKRLIYAA..
.....SSLESGVP.SRFSGSG..SGTDFTLTISSLQPEDFATYYCLQYNSDP
>AY963728|IGKV1-43*03|Macaca mulatta|F|V-REGION|169..455|287 nt|1| | | |95 AA|95+17=112| | |
DIQMTQSPSSLSASVGDRVT.ITCRASQGI......STYLNWYQQKPGKAPKLLIYAA..
.....SSLESGVP.SRFSGSG..SGTEFTLTISSLQPEDFATYYCLQYNSNP
>IMGT000063|IGKV1-44*01|Macaca mulatta_AG07107|F|V-REGION|678108..678394|287 nt|1| | | |95 AA|95+17=112| | |
DIQMTQSPSSLSASVGDRVT.ITCRASQTI......SSYLAWYQQKPGKVPKLLIYAA..
.....SSLESGVP.SRFSGSG..SGTEFTLTISSLQPEDFATYYCQQHNSHP
>NW_001099007|IGKV1-44*02|Macaca mulatta_17573|F|V-REGION|720888..721174|287 nt|1| | | |95 AA|95+17=112| |rev-compl|
DIQMTQSPSSLSASVGDRVT.ITCRASQTI......SSYLAWYQQKPGKVPKLLIYAA..
.....SSLESGVP.SRFSGSG..SGTEFTLTISSLQPEDFATYYCQQHNSHP
>AY963718|IGKV1-44*03|Macaca mulatta|F|V-REGION|159..445|287 nt|1| | | |95 AA|95+17=112| | |
DIQMTQSPSSLSASVGDRVT.ITCRASQTI......SSYLAWYQQKPGKVPKLLIYAA..
.....STLQSGVP.SRFSGSG..SGTEFTLTISSLQPEDFATYYCQQHNSHP
>IMGT000063|IGKV1-46*01|Macaca mulatta_AG07107|F|V-REGION|662087..662373|287 nt|1| | | |95 AA|95+17=112| | |
DIQMTQSPSSLSASVGDSVT.ITCRASQSF......SSSLAWYQQKPGKAPKLLIYSA..
.....SSLQSGVP.SRFSGSK..SGTDFTLTISSLQPEDIASYYCQQYYSDP
>IMGT000063|IGKV1-52*01|Macaca mulatta_AG07107|ORF|V-REGION|582014..582300|287 nt|1| | | |95 AA|95+17=112| | |
DIQMTQAPSFLSASVGDRVT.ITCQASQSI......SNDLAWYQQKPGKSPTLLIYDA..
.....TNLHTGVP.SRFSGSG..SGADYSLTISSLELEDFAAYYCQQEYSYP
>IMGT000063|IGKV1-56*01|Macaca mulatta_AG07107|P|V-REGION|536476..536762|287 nt|1| | | |95 AA|95+17=112| | |
DFQMIQSPSSLTTSLEDRVT.ITCWASQGI......CNYLS*YHQKLGNPPQLLIYAA..
.....PSLQSGVP.SRFSSSR..SGTHFTLTISSLQPEDFITYCCLQTYSSH
>NW_001099007|IGKV1-56*02|Macaca mulatta_17573|P|V-REGION|862358..862644|287 nt|1| | | |95 AA|95+17=112| |rev-compl|
DFQMIQSPSSLTTSLGDRVT.ITCWASQGI......CNYLS*YHQKLGNPPQLLIYAA..
.....PSLQSGVP.SRFSSSR..SGTHFTLTISSLQPEDFITYCCLQTYSSH
>IMGT000063|IGKV1-59*01|Macaca mulatta_AG07107|F|V-REGION|495200..495486|287 nt|1| | | |95 AA|95+17=112| | |
AIQMTQSPSSLSASVGDKVT.ITCRASQSI......GSNLAWYQQKPGKVPKLLIYAA..
.....STLQSEVP.SRFSGSG..SGTDFTLTISSLQPEEVATYYCQKCDSAP
>NW_001099007|IGKV1-59*02|Macaca mulatta_17573|F|V-REGION|903554..903840|287 nt|1| | | |95 AA|95+17=112| |rev-compl|
AIQMTQSPSSLSASVGDTVT.ITCRASQSI......GSNLAWYQQKPGKVPKLLIYAA..
.....STLQSEVP.SRFSGSG..SGTDFTLTISSLQPEEVATYYCQKCDSAP
>IMGT000063|IGKV1-6*01|Macaca mulatta_AG07107|F|V-REGION|1257951..1258237|287 nt|1| | | |95 AA|95+17=112| |rev-compl|
DIQMTQSPTFLSASVGDRVT.IACRASQSI......HKGLAWYQQKPGKAPKLLISYA..
.....SNLYTDVP.SRFSGSG..SGTDFTLTISSLQPEDAATYYCFQYNSYP
>IMGT000063|IGKV1-66*01|Macaca mulatta_AG07107|F|V-REGION|415961..416246|286 nt|1| | | |95 AA|95+17=112| | |
DIQMTQSPSSLSASVGDRVT.ITCRASQGI......NNYLSWYQQKPGKAPKPLIYYA..
.....SSLERGVP.SRFSGSR..SGTDYTLTISSLQPEDIATYYCQQYIIPL
>IMGT000063|IGKV1-69*01|Macaca mulatta_AG07107|F|V-REGION|386391..386677|287 nt|1| | | |95 AA|95+17=112| | |
DIQMTQSPSSLSASVGDRVT.ITCRASQGI......SNWLAWYQQKPGKAPKLLIYRA..
.....SNLETGVP.SRFSGSG..SGTDFTLTISSLQPEDIATYYCQQHDNSP
>AY963736|IGKV1-69*02|Macaca mulatta|F|V-REGION|166..452|287 nt|1| | | |95 AA|95+17=112| | |
DIQMTQSPSSLSASVGDRVT.ITCRASQGI......SNWLAWYQQKPGKAPKLLIYAA..
.....SNLETGVP.SRFSGSG..SGTDFTLTISSLQPEDIATYYCQQHDNSP
>IMGT000063|IGKV1-74*01|Macaca mulatta_AG07107|F|V-REGION|324838..325124|287 nt|1| | | |95 AA|95+17=112| | |
DIQMTQSPSSLSASVGDRVT.ITCRASENV......NNYLNWYQQKPGKAPKLLIYKA..
.....STLQSGVP.SRFSGSG..SGTDYTFTISSLQPEDVATYYCQHGYGTP
>IMGT000063|IGKV1-80*01|Macaca mulatta_AG07107|F|V-REGION|282335..282621|287 nt|1| | | |95 AA|95+17=112| | |
DIQMTQSPSSLSASVGDTVT.ITCRASQGI......NNELAWYQQKPGKAPTLLLYSG..
.....SSLHTGVP.SQFSGSG..SGTDFTLTISSLQPEDVATYYCRQDYNTP
>IMGT000063|IGKV1-84*01|Macaca mulatta_AG07107|F|V-REGION|260250..260536|287 nt|1| | | |95 AA|95+17=112| | |
DIQMTQPPSSLSASVGDRVN.ITCQASQSI......SNYLNWYPQKTWKAPKFLTYRA..
.....SGLQRGVP.SQFSGSG..YGRDFTLTISSLRPEDFAIYYCQQESIFP
>IMGT000063|IGKV1-94*01|Macaca mulatta_AG07107|F|V-REGION|184522..184808|287 nt|1| | | |95 AA|95+17=112| | |
DIQMTQSPSSLSASVGDRVT.VTCRASQGI......NKELSWYQQKPGKAPTLLIYAA..
.....SSLQTGVS.SRFSGSG..SGTDFTLTISSLQPEDVATYYCQQDYTTP
>AY963719|IGKV1S11*01|Macaca mulatta|F|V-REGION|168..454|287 nt|1| | | |95 AA|95+17=112| | |
DIQMTQSPSSLSASVGDTVT.ITCRASQGI......SSWLAWYQQKPGKAPKLLIYAA..
.....SSLQSGVP.SRFSGSG..SGTDFTLTISSLQPEDFAVYYCQQRNSYP
>AY963720|IGKV1S12*01|Macaca mulatta|F|V-REGION|159..445|287 nt|1| | | |95 AA|95+17=112| | |
DIQMTQSPSSLSASVGDRVT.ITCRASQTI......SSYLAWYQQKPGKVPKLLIYAA..
.....STLQSGVP.SRFSGSG..SGTEFTLTISSLQPEDFATYYCLQYNSNP
>AY963721|IGKV1S13*01|Macaca mulatta|F|V-REGION|167..453|287 nt|1| | | |95 AA|95+17=112| | |
DIQMTQSPSSLSASVGDTVT.ITCRASQGI......SNYLAWYQQKPGKAPKLLIYAA..
.....SSLESGVP.SRFSGSG..SGTDFTLTISSLQPEDFAAYYCLQHNSYP
>AY963722|IGKV1S14*01|Macaca mulatta|F|V-REGION|168..454|287 nt|1| | | |95 AA|95+17=112| | |
DIQMTQSPSSLSASVGDTVT.ITCRASQGI......SNYLAWYQQKPGKAPKPLIYYA..
.....SNLESGVP.SRFSGSG..SGTDFTLTISSLQPEDFATYYCQQHNSYP
>AY963723|IGKV1S15*01|Macaca mulatta|F|V-REGION|168..454|287 nt|1| | | |95 AA|95+17=112| | |
DIQMTQSPSSLSASVGDTVT.ITCRASQGI......SNYLAWYQQKPGKAPKPLIYYA..
.....SSLESGVP.SRFSGSG..SGTDFTLTISSLQPEDFATYYCQQGNSNP
>AY963724|IGKV1S16*01|Macaca mulatta|F|V-REGION|168..454|287 nt|1| | | |95 AA|95+17=112| | |
DIQMTQSPSSLSASVGDTVT.ITCRASQGI......SNYLAWYQQKPGKAPKPLIYYA..
.....SSLESGVP.SRFSGSG..YGTDFTLTISSLQPEDFATYYCQQHNSYP
>AY963725|IGKV1S17*01|Macaca mulatta|F|V-REGION|168..454|287 nt|1| | | |95 AA|95+17=112| | |
DIQMTQSPSSLSASVGDRVT.ITCRASQGI......SNNLAWYQQKPGKAPKLLIYAA..
.....SSLESGVP.SRFSGSG..SGTDFTLTISSLQPEDFATYYCLQYNSNP
>AY963727|IGKV1S19*01|Macaca mulatta|F|V-REGION|168..454|287 nt|1| | | |95 AA|95+17=112| | |
DIQMTQSPSSLSASVGDTVT.ITCRASQGI......SNYLNWYQQKPGKAPKLLIYAA..
.....SSLESGVP.SRFSGSG..SGTDFTLTISSLQPEDFAAYYCLQHNSYP
>AY963729|IGKV1S21*01|Macaca mulatta|F|V-REGION|168..454|287 nt|1| | | |95 AA|95+17=112| | |
DIQMTQSPSSLSASVGDRVT.ITCRASQGI......SNYLNWYQQKPGKAPKRLIYDA..
.....SSLQSGVP.SRFSGSG..SGTDFTLTISSLQPEDFAAYYCLQYNSAP
>AY963733|IGKV1S25*01|Macaca mulatta|F|V-REGION|168..454|287 nt|1| | | |95 AA|95+17=112| | |
DIQMTQSPSSLSASVGDRVT.ITCRASQGI......SSWLAWYQQKPGKVPKLLIYAA..
.....STLQSGVP.SRFSGSG..SGTEFTLTISSLQPEDFATYYCQQHNSHP
>AY963711|IGKV1S3*01|Macaca mulatta|F|V-REGION|159..445|287 nt|1| | | |95 AA|95+17=112| | |
DIQMTQSPSSLSASVGDTVT.ITCRASQGI......SNYLAWYQQKPGKAPKPLIYYA..
.....SSLESGVP.SRFSGSG..SGTDFTLTISSLQPEDFATYYCQQHNSYP
>AY963712|IGKV1S4*01|Macaca mulatta|F|V-REGION|168..454|287 nt|1| | | |95 AA|95+17=112| | |
DIQMTQSPSSLSASVGDRVT.ITCQASQGI......SSWLAWYQQKPGKAPKLLLYKA..
.....PGLQSGVP.SMFSGSG..SGTEFTLTISSLQPEDFATYYCQQHNSYP
>AY963713|IGKV1S5*01|Macaca mulatta|F|V-REGION|159..445|287 nt|1| | | |95 AA|95+17=112| | |
DIQMTQSPSSLSASVGDRVT.ITCRASQTI......SSYLAWYQQKPGKAPKRLIYAA..
.....SSLESGVP.SRFSGSG..SGTEFTLTISSLQPEDFATYYCQQHNSHP
>AY963714|IGKV1S6*01|Macaca mulatta|F|V-REGION|168..454|287 nt|1| | | |95 AA|95+17=112| | |
DIQMTQSPSSLSASVGDKVT.ITCRASQGI......SSWLAWYQQKPGKAPKLLIYKA..
.....SSLASGVP.SRFSGSG..SGTEFTLTISSLQPEDFATYYCQQHNSYP
>AY963716|IGKV1S8*01|Macaca mulatta|F|V-REGION|159..445|287 nt|1| | | |95 AA|95+17=112| | |
DIQMTQSPSSLSASVGDRVT.ITCRASQTI......SSYLAWYQQKPGKVPKLLIYAA..
.....STLQSGVP.SRFSGSG..SGTDFTLTISSLQPEDFATYYCQQYNSAP
>AY963717|IGKV1S9*01|Macaca mulatta|F|V-REGION|168..454|287 nt|1| | | |95 AA|95+17=112| | |
DIQMSQSPSSLSASVGDTVT.ITCRASQGI......SNYLNWFQQKPGKAPKLLIYAA..
.....TTLQSGVP.SRFSGSG..SGTDFTLTISSLQPEDFATYYCQQHNSYP
>IMGT000063|IGKV2-103*01|Macaca mulatta_AG07107|P|V-REGION|18428..18730|303 nt|1| | | |101 AA|101+12=113| |rev-compl|
DILLTQTPLSLPVTPGEPAS.ISCRSSQSLLHS.NRNTYLDWYLQKPGQSPQLLIYEV..
.....SKRASGVP.ERFSGSG..SDTDFTLKINLVEAEDVGVYYCMQTTQ*PP
>IMGT000063|IGKV2-104*01|Macaca mulatta_AG07107|F|V-REGION|13545..13849|305 nt|1| | | |101 AA|101+11=112| |rev-compl|
DIVMTQTPLSLPVTLGEPAS.ISCRSSQSLLDSEDGNTYLEWYLQKPGQSPQLLIYEV..
.....SNRASGVP.DRFSGSG..SDTDFTLKISRVEAEDVGVYYCMQALEFP
>AY963756|IGKV2-104*02|Macaca mulatta|F|V-REGION|412..716|305 nt|1| | | |101 AA|101+11=112| | |
DIVMTQTPLSLPVTPGEPAS.ISCRSSQSLLDSEDGNTYLDWYLQKPGQSPQLLIYEV..
.....SNRASGVP.DRFSGSG..SDTDFTLKISRVEAEDVGVYYCMQALEFP
>NW_001099007|IGKV2-13-1*01|Macaca mulatta_17573|F|V-REGION|252813..253114|302 nt|1| | | |100 AA|100+12=112| |rev-compl|
DIVMIQNPLSLPVPPGQPAS.ISCRSSQSFLHS.DESTYLDWYLQKPGQSPWLLIYLI..
.....SHKFYGVP.SEFSGSR..SGTGFTPKFSKVEAEDVGVYCCEQGLQGP
>IMGT000063|IGKV2-2*01|Macaca mulatta_AG07107|P|V-REGION|1309557..1309855|299 nt|1| | | |99 AA|99+13=112| | |
VIVMTRIPLSLPVTPGEPAS.ISYRSS*SLLHS.DEYTYLHWYLRKPG*FPQLLIYGV..
.....SN.QFWLP.DRFSGSE..LGTDFTLKVSRVEVEDVGIYHYLQGTQSL
>NW_001099007|IGKV2-2*02|Macaca mulatta_17573|P|V-REGION|102941..103239|299 nt|1| | | |99 AA|99+13=112| |rev-compl|
VIVMTRIPLSLPVTPGEPAS.ISYKSS*SLLHS.DEYTYLHWYLQKPG*FPQLLIYGV..
.....SN.QFWLP.DRFSGSE..LGTDFTLKVSRVEVEDVGIYHYLQGTQSL
>IMGT000063|IGKV2-23*01|Macaca mulatta_AG07107|P|V-REGION|1026066..1026367|302 nt|1| | | |100 AA|100+12=112| | |
DAVMTQPPLSLPITPSEPAS.ISYRSSHSPLHS.NGCTYLNWDLQKPG*PPWLPIYLI..
.....SNQDPGVL.DRFSGSG..SVADFMLKISRVDAEDVGVYYCQQSTHYP
>NW_001099007|IGKV2-23*02|Macaca mulatta_17573|P|V-REGION|341251..341522|272 nt|1| | ||90 AA|90+22=112|partial in 5'|rev-compl|
..........LPITPSEPAS.ISYRSSHSPLHS.NGCTYLNWDLQKPG*PPWLPIYLI..
.....SNQDPGVL.DRFSGSG..SVADFMLKISTVDAEDVGVYYCQQSTHYS
>IMGT000063|IGKV2-26*01|Macaca mulatta_AG07107|P|V-REGION|986747..987048|302 nt|1| | | |100 AA|100+12=112| | |
NAVMSQPPLSLPVTPGEPAS.NSYRSSHSPLHS.NGYTYLNWDLRKPGQPPRLPIYLV..
.....SNQDPGVT.DMFSGRE..SGTDFMLKISRVEAEDVGVYYCQQNTFFR
>NW_001099007|IGKV2-26*02|Macaca mulatta_17573|P|V-REGION|403420..403721|302 nt|1| | | |100 AA|100+12=112| |rev-compl|
DAVMSQPPLSLPVTPGEPGS.ISYRSSHSPLHS.NGCTFLNWDLQKPGQPPRLLIYLI..
.....SNRDPGVP.DTFSGRE..SGTDFMLKISRVEAEDVGVYYCQQNTFFR
>IMGT000063|IGKV2-30*01|Macaca mulatta_AG07107|P|V-REGION|920135..920435|301 nt|1| | | |100 AA|100+12=112| | |
DAVMTQLPLCLPVTLGELAS.VSYRLSHSPLHS.NGYTYLK*YLQKPGQPPRLLIYLV..
.....SNWDPGVS.DRFNGSV..SVTDFMLKITRVDIEDIGVYYCQQSTH*R
>NW_001099007|IGKV2-30*02|Macaca mulatta_17573|P|V-REGION|470585..470885|301 nt|1| | | |100 AA|100+12=112| |rev-compl|
DAVMTQLPLCLPVTLGELAS.VSYRLSHSPLHS.NGYTYLK*YLQKPGQPPRLLICLV..
.....SNWDPGVS.DRFNGSV..SVTDFMLKITRVDIEDIGVYYCQQSTH*R
>IMGT000063|IGKV2-34*01|Macaca mulatta_AG07107|P|V-REGION|829837..830138|302 nt|1| | | |100 AA|100+12=112| | |
AVVMTQPPLSLPVTPEEPYS.ISCRPSHSPLHS.NGYTYLNWVLQKPGQPPWLPIYWV..
.....SNREPGVP.ERFSGSE..SVINFRLKIRRMDAEDVEVYCCQQSTHYS
>IMGT000063|IGKV2-39*01|Macaca mulatta_AG07107|P|V-REGION|750912..751216|305 nt|1| | | |101 AA|101+12=113| | |
DTVMTQPPLSLPITTGEPGS.ISFRPSHSPLHS.NGYTYLNWDLQNPGQPPWLPIYLV..
.....SNRNPGVP.KRFRDSE..SMTTFRFKIRRMDAEDVEVYCCCQQSTHYP
>IMGT000063|IGKV2-48*01|Macaca mulatta_AG07107|P|V-REGION|631520..631821|302 nt|1| | | |100 AA|100+12=112| | |
DIVMIQTPLSLPVTPGEPAS.ISCRSNQNLLHS.DGNTYLHWYLQKPGQSPQLLIYGG..
.....SNRFAGVP.DRFSGSG..SGTDFTLKISQVEAEDVGVY*CTQGIQFP
>NW_001099007|IGKV2-48*02|Macaca mulatta_17573|P|V-REGION|766915..767216|302 nt|1| | | |100 AA|100+12=112| |rev-compl|
DIVMIQTPLSLPVTPGEPAS.ISCRSNQNLLHS.DGNTYLHWYLQKPGQSPQLLIYGG..
.....SNRFAGVP.DRFSGSG..SGTDFTLKISQVEAEDVGVY*CTQGIQFP
>IMGT000063|IGKV2-49*01|Macaca mulatta_AG07107|P|V-REGION|618913..619214|302 nt|1| | | |100 AA|100+12=112| | |
DIELTQTPHSLPVNPGEPAS.ISC*SSQSLLHS.DGKTYFYWFLQKPGQHPRHPIYWV..
.....SNWNSGAP.DRFSGSG..AGTDFTLRIGVVEAEDVGVYYYMQVTHWP
>NW_001099007|IGKV2-49*02|Macaca mulatta_17573|P|V-REGION|779602..779903|302 nt|1| | | |100 AA|100+12=112| |rev-compl|
DIELTQTPHSLPVNPGEPAS.ISC*SSQSLLHS.DGKTYFYWFLQKPGQHPRHPIYWV..
.....SNWNSGAP.DRFSGSG..AGTDFTLRIGVVEAEDVGVYYYMQVTHWP
>IMGT000063|IGKV2-58*01|Macaca mulatta_AG07107|F|V-REGION|530936..531237|302 nt|1| | | |100 AA|100+12=112| | |
DVAMTQSPLSLPVTLGQPAS.ISCRSSQSLLHS.NGNTYLSWFQQKPGQSPRRLIYKV..
.....SNRDSGVP.DRFSGSG..AGTDFTLKISRVEAEDVGVYYCMEGTHHP
>NW_001099007|IGKV2-58*02|Macaca mulatta_17573|P|V-REGION|867900..868201|302 nt|1| | | |100 AA|100+12=112| |rev-compl|
DVAMTQSPLSLPVTPGQPAS.ISCRSSQSLLHS.NGNTYLS*FQQKPGQSPRRLIYKV..
.....SNRDSGVP.DRFSGSG..AGTDFTLKISRVEAEDVGVYYCMEGTHHP
>AY963743|IGKV2-58*03|Macaca mulatta|F|V-REGION|464..765|302 nt|1| | | |100 AA|100+12=112| | |
DVAMTQSPLSLPVTPGQPAS.ISCRSSQSLLHS.NGNTYLSWFQQKPGQSPRRLIYKV..
.....SNRDSGVP.DRFSGSG..AGTDFTLKISRVEAEDVGVYYCMEGTHHP
>IMGT000063|IGKV2-60*01|Macaca mulatta_AG07107|F|V-REGION|486913..487214|302 nt|1| | | |100 AA|100+12=112| | |
DIVMTQTPLSLPVTLGEPAS.ISCRSSQSLLSS.NGYNYLNWYLQKPGQSPQLLIYYG..
.....SNRASGVP.DRFSGSG..SGTDFTLKISRVEAEDVGVYYCMQALQTP
>NW_001099007|IGKV2-60*02|Macaca mulatta_17573|F|V-REGION|911803..912104|302 nt|1| | | |100 AA|100+12=112| |rev-compl|
DIVMTQTPLSLPVTLGEPAS.ISCRSSQSLLSS.NGYNYLNWYLQKPGQSPQLLIYYG..
.....SNRASGVP.DRFSGSG..SGTDFTLKISRVEAEDVGVYYCMQALQTP
>IMGT000063|IGKV2-61*01|Macaca mulatta_AG07107|F|V-REGION|462634..462935|302 nt|1| | | |100 AA|100+12=112| | |
DIVMTQTPLSLPVTPGEPAS.ISCRSSQSLLHT.DGYTYLDWYLQKPGQSPQLLIYGG..
.....SNRASGVP.DRFSGSG..SGTDFTLKISKVEAEDVGVYYCMQHKALP
>AY963751|IGKV2-61*03|Macaca mulatta|F|V-REGION|400..701|302 nt|1| | | |100 AA|100+12=112| | |
DIVMTQTPLSLPVTPGEPAS.ISCRSSQSLLHT.DGYTYLDWYLQKPGQSPQLLIYGG..
.....SNRASGVP.DRFSGSG..SGTDFTLKISKVEAEDVGVYYCMQHKALP
>NW_001099007|IGKV2-62*02|Macaca mulatta_17573|F|V-REGION|946290..946591|302 nt|1| | | |100 AA|100+12=112| |rev-compl|
DVVMTQSPLSLPITPGQPAS.ISCRSSQSFVHS.DGNTYLSWYQQKPGQPPRLLIYKV..
.....SNRDSGVP.DRFSGSG..AGTDFTLKISRVEAEDVGVYYCGQGTKVP
>IMGT000063|IGKV2-64*01|Macaca mulatta_AG07107|F|V-REGION|438026..438327|302 nt|1| | | |100 AA|100+12=112| | |
DVVMTQSPLSLPITPGQPAS.ISCRSSQSLVHS.DGNTYLSWYQQKPGQPPRLLIYKV..
.....SNRDSGVP.DRFSGSG..AGTDFTLKISRVEAEDVGVYYCGQGTHWP
>AY963742|IGKV2-64*02|Macaca mulatta|F|V-REGION|466..766|301 nt|1| | | |100 AA|100+12=112| | |
DVVMTQSPLSLPITPGQPAS.ISCRSSQSLVHS.DGNTYLSWYQQKPGQPPRLLIYKV..
.....SNRYSGVP.DRFSGSG..AGTDFTLKISRVEAEDVGVYYCGQGTHCL
>IMGT000063|IGKV2-65*01|Macaca mulatta_AG07107|F|V-REGION|421726..422027|302 nt|1| | | |100 AA|100+12=112| | |
DVVMTQSPLSLPITPGQPAS.ISCRSSQSLVHS.NGNTYLSWYQQKPGQPPRRLIYEV..
.....SNRDSGVP.DRFSGSG..AGTDFTLKISRVEAEDVGVYYCGQGTHLP
>IMGT000063|IGKV2-68*01|Macaca mulatta_AG07107|F|V-REGION|394365..394665|301 nt|1| | | |100 AA|100+12=112| | |
DIVMTQTLLSLPVTPGEPAS.ISCRSSQSLLHS.NGNTYLDWYLQKPGQSPRFLIYKV..
.....TNREPGVP.DRFSGSG..SGTDFTLKISRVEPEDVGVCYCMQSTKDP
>IMGT000063|IGKV2-7*01|Macaca mulatta_AG07107|P|V-REGION|1229225..1229520|296 nt|1| | | |98 AA|98+14=112| |rev-compl|
DTVMTQTPLSLPVT.G.PAS.ISCRSSQSLPYG.NGVNYLNWYLQKPDQPPQLLIYLG..
.....SSRFPGVP.DRFTVSR..SDTDFTLQISRVKAEDVGVYY*VQCLQLP
>AY963738|IGKV2-7*02|Macaca mulatta|F|V-REGION|435..730|296 nt|1| | | |98 AA|98+14=112| | |
DTVMTQTPLSLPVT.G.PAS.ISCRSSQSLPYG.NGVNYLNWYLQKPDQPPQLLIYLG..
.....SSRFPGVP.DRFTVSR..SDTDFTLQISRVKAEDVGVYYCVQCLQLP
>IMGT000063|IGKV2-70*01|Macaca mulatta_AG07107|F|V-REGION|364525..364826|302 nt|1| | | |100 AA|100+12=112| | |
DIVMTQTPLSLSVTPREPAS.ISCRSSQSLLHT.DGRTYLYWYLQKPGQPPRLLIYRV..
.....SNRFSGVP.DRFSGSG..SGTDFTLKISRVEAEDVGVYYCMQALQTP
>IMGT000063|IGKV2-72*01|Macaca mulatta_AG07107|F|V-REGION|345531..345832|302 nt|1| | | |100 AA|100+12=112| | |
DIVMTQTPLSLPITPGEPAS.ISCRSSQSLLHS.NGNTYLHWYLQKPGQSPQLLIYGG..
.....SNRASGVP.DRFSGSG..SGTDFTLKISKVEAEDVGVYYCVQAIAFP
>AY963748|IGKV2-72*02|Macaca mulatta|F|V-REGION|401..702|302 nt|1| | | |100 AA|100+12=112| | |
DIVMTQTPLSLPITPGEPAS.ISCRSSQSLLHS.NGNTYLHWYLQKPGQSPQLLIYGG..
.....SNRASGVP.DRFSGSG..SGTDFTLKISKVEAEDVGVYYCVQAIAFP
>AY963749|IGKV2-72*03|Macaca mulatta|F|V-REGION|401..702|302 nt|1| | | |100 AA|100+12=112| | |
DIVMTQTPLSLPITPGEPAS.ISCRSSQSLLHS.NGNTYLHWYLQKPGQSPQLLIYGG..
.....SNRASGVP.DRFSGSG..SGTDFTLKISKVEAEDVGVYYCVHAIAFP
>IMGT000063|IGKV2-73*01|Macaca mulatta_AG07107|F|V-REGION|340537..340838|302 nt|1| | | |100 AA|100+12=112| | |
DIVMTQTPLSLPVTPGEPAS.ISCRSSQSLLHS.DGNTYLYWYLQKPGQPPRLLIYRV..
.....SNRFSGVP.DRFSGSG..SGTDFTLKISRVKAEDVGVYYCMQALQTP
>IMGT000063|IGKV2-75*01|Macaca mulatta_AG07107|P|V-REGION|315641..315942|302 nt|1| | | |100 AA|100+12=112| | |
DIVMTKTPLSWPVSAGETAP.ISCRSSQSLLPN.DGYSYLY*FLQTLGESPQLLIYFV..
.....SNRASGVS.DRFSDSD..SSTDFTLKIRRVEAEDVGVYYCMEVL*FP
>IMGT000063|IGKV2-76*01|Macaca mulatta_AG07107|F|V-REGION|311626..311930|305 nt|1| | | |101 AA|101+11=112| | |
DIVMTQTPLSLPITPGEPAS.ISCRSSQSFLDSDDGYTYLDWYLQKPGQPPQPLIYFV..
.....SSRASGVP.DRFNGSG..SGSDFTLKISGVEADDVGVYYCMQCIEFI
>IMGT000063|IGKV2-78*01|Macaca mulatta_AG07107|F|V-REGION|290433..290734|302 nt|1| | | |100 AA|100+12=112| | |
DIVMTQTPLSLPVTPGEPAS.ISCRSSQSLLDS.DGYTHLHWYLQKPGQSPQLLIYLG..
.....SNRASGVP.DRFSGSG..SGTDFTLKISRVEAEDVGVYYCMQTLQTP
>IMGT000063|IGKV2-81*01|Macaca mulatta_AG07107|ORF|V-REGION|277659..277960|302 nt|1| | | |100 AA|100+12=112| | |
DIVMTQTPLSLPVIPGEPAS.ISCRSSQSLLHS.DGYTYLHWYLQKPGQSPQLLIYLG..
.....SNCASGVP.DRFSGSG..SGTDFTLKISRVEAEDIGIYYRMQNLQIP
>IMGT000063|IGKV2-82*01|Macaca mulatta_AG07107|F|V-REGION|274224..274525|302 nt|1| | | |100 AA|100+12=112| | |
DIVMTQTPLSLPVTLGEPAS.ISCRSSQSLVYS.DGKTYLDWYLQKPGQSPQLLMYLV..
.....SKRASGVP.DKFSGSG..SGTDFTLKISRVEAEDVGVYYCMQALRSP
>AY963757|IGKV2-82*02|Macaca mulatta|F|V-REGION|424..725|302 nt|1| | | |100 AA|100+12=112| | |
DIVMIQTPLSLPVTLGEPAS.ISCRSSQSLVYS.DGKTYLYWYLQKPGQSPQLLMYLV..
.....SKRASGVP.DKFSGSG..SGTDFTLKISRVEAEDVGVYYCMQALRSP
>AY963759|IGKV2-82*03|Macaca mulatta|F|V-REGION|425..726|302 nt|1| | | |100 AA|100+12=112| | |
DIVMIQTPLSLPVTLGEPAS.ISCRSSQSLVYS.DGKTYLYWYLQKPGQSPQLLMYLV..
.....SKRASGVP.DKFSGSG..SGTDFTLKISRVEAEDVGVYYCMQALRSP
>IMGT000063|IGKV2-83*01|Macaca mulatta_AG07107|P|V-REGION|269349..269650|302 nt|1| | | |100 AA|100+12=112| | |
DILLTQTPLSLPVTPGEPAS.ISCRSSQSLLHS.NGNAYLHCYLQKPGQSPQLLIYEV..
.....SNRASGVP.DRFSGSG..SDTDFTLKINLVEAEDVGVYSCMQTTQ*P
>IMGT000063|IGKV2-86*01|Macaca mulatta_AG07107|F|V-REGION|251568..251872|305 nt|1| | | |101 AA|101+11=112| | |
DIVMTQTPLSLPVTPGEPAS.ISCRSSQSLLDSEDGNTYLDWYLQKPGQSPQPLIYEV..
.....SNRASGVP.DRFSGSG..SDTDFTLKISRVEAEDVGVYYCMQYTHIP
>AY963753|IGKV2-86*02|Macaca mulatta|ORF|V-REGION|415..719|305 nt|1| | | |101 AA|101+11=112| | |
DIVMTQTPLSLPVTPGEPAS.ISRRSSQSLLDSEDGNTYLDWYLQKPGQSPQPLIYEV..
.....SNRASGVP.DRFSGSG..SDTDFTLKISRVEAEDVGVYYCMQYTHIP
>IMGT000063|IGKV2-87*01|Macaca mulatta_AG07107|ORF|V-REGION|243377..243681|305 nt|1| | | |101 AA|101+11=112| | |
DIVMTQTPLSLLVTPGEPASVSSCRCSQSLPYS.GEKTYLYWYLQKPGQSPQLLIYFV..
.....SNRASQVP.DRFRGSG..SDTDFTLKISWVETEDVGIYYRMQNIELP
>IMGT000063|IGKV2-90*01|Macaca mulatta_AG07107|F|V-REGION|220954..221255|302 nt|1| | | |100 AA|100+12=112| | |
DIVMTQTPLSLPVTPGEPAS.ISCRSSQSLLDS.DGYTCLDWYLQKPGQSPQLLIYEV..
.....SNRVSGVP.DRFSGSG..SGTDFTLKISRVEAEDVGVYYCMQSIEFP
>IMGT000063|IGKV2-91*01|Macaca mulatta_AG07107|F|V-REGION|203418..203719|302 nt|1| | | |100 AA|100+12=112| | |
DIVMTQTPLSLPVTPGEPAS.ISCRSSQSLLHS.NGYTYLYWYLQKPGQSPQLLMYFA..
.....SYRASGVP.DRFSGSG..SGTDFTLGISRVEAEDIGVYYCMQGTQLP
>IMGT000063|IGKV2-99*01|Macaca mulatta_AG07107|F|V-REGION|127811..128115|305 nt|1| | | |101 AA|101+11=112| | |
DIVMTQTPLSLPVTPGEPAS.ISCRSSQSLFDSDYANTYLDWCLQKPGQSPQLLIYML..
.....FNRVSGVP.DRFSGSG..SGTDFTLKISRVEAEDVGVYYCMQSIEYP
>AY963754|IGKV2S15*01|Macaca mulatta|F|V-REGION|404..705|302 nt|1| | | |100 AA|100+12=112| | |
DIVMTQTPLSLSVTPGQPAS.ISCKSSQSLLHS.DGKTYLYWYLQKPGQSPQLLIYEV..
.....SSRFSGVP.DRFSGSG..SGTDFTLKISRVEAEDVGVYYCMQGIHLP
>AY963755|IGKV2S16*01|Macaca mulatta|ORF|V-REGION|415..719|305 nt|1| | ||101 AA|101+11=112| | |
DIVMTQTPLSLPVTPGEPAS.ISCRSSQSLLDSEDGNTYLDRYLQKPGQSPQLLIYEV..
.....SNRASGVP.DRFSGSG..SDTDFTLKISRVEAEDVGVYYCMQGIEYP
>AY963739|IGKV2S2*01|Macaca mulatta|F|V-REGION|442..742|301 nt|1| | | |100 AA|100+12=112| | |
DIVMTQTPLSLPVTPGEPAS.ISCRSSQSLLHS.NGNTYLDWYLQKPGQSPRLLIYKV..
.....TNRESGVP.DRFSGSG..SGTDFTLKISRVEPEDVGVYYCMQSTKDP
>AY963760|IGKV2S20*01|Macaca mulatta|F|V-REGION|415..719|305 nt|1| | ||101 AA|101+11=112| | |
DIVMTQTPLSLPVTPGEPAS.ISCRSSQSLLDSEDGNTYLEWYLQKPGQSPQPLIYEV..
.....SNRASGVP.DRFSGSG..SDTDFTLKISRVEAEDVGVYYCMQGIEYP
>AY963740|IGKV2S3*01|Macaca mulatta|F|V-REGION|442..742|301 nt|1| | | |100 AA|100+12=112| | |
DIVMTQTPLSLPVTPGEPAS.ISCRSSQSLLHS.NGNTYLHWYLQKPGQSPRLLIYKV..
.....TNRESGVP.DRFSGSG..SGTDFTLKISRVEPEDVGVYYCMQSTKDP
>AY963746|IGKV2S8*01|Macaca mulatta|F|V-REGION|467..767|301 nt|1| | | |100 AA|100+12=112| | |
DVVMTQSPLSLPVTPGQPAS.ISCRSSQSLVHS.DGKTYLNWLQQKPGQPPRRLIYQV..
.....SNRDSGVP.DRFSGSG..AGTDFTLKISRVEAEDVGVYYCGQGTHVL
>AY963747|IGKV2S9*01|Macaca mulatta|F|V-REGION|467..767|301 nt|1| | | |100 AA|100+12=112| | |
DVVMTQSPLSLPVTPGQPAS.ISCRSSQSLVHS.DGKTYLNWLQQKPGQPPRRLIYQV..
.....SNRDSGVP.DRFSGSG..AGTDFTLKISRVEAEDVGVYYCVQGTHCL
>IMGT000063|IGKV3-10*01|Macaca mulatta_AG07107|F|V-REGION|1211190..1211476|287 nt|1| | | |95 AA|95+17=112| |rev-compl|
QVILTQSPATLSLSPGERAT.LSCRASQSV......SSYLAWYQQKPGQAPRLLIHSA..
.....SSRATGIP.DRFSGSG..SGTEFTLTISSLEPEDVGVYHCYQHSSGY
>NW_001099007|IGKV3-10*02|Macaca mulatta_17573|F|V-REGION|201688..201974|287 nt|1| | | |95 AA|95+17=112| | |
QVILTQSPATLSLSPGERAT.LSCRASQSV......SSYLAWYQQKPGQAPRLLIHSA..
.....SSRATGIP.DRFSGSG..SGTEFTLTISSLEPEDVGVYHCYQYYSGY
>IMGT000063|IGKV3-12*01|Macaca mulatta_AG07107|P|V-REGION|1194603..1194892|290 nt|1| | | |96 AA|96+16=112| | |
EIVLTQSPTSMAVSQGERVT.ISCTASSSVS.....TSYLHWYQQKPGFPPRLLVYRT..
.....SSLASGVP.ARFSGSG..SGTSYTLTISSMEAEDAANYYCQQGNSIP
>NW_001099007|IGKV3-12*02|Macaca mulatta_17573|P|V-REGION|218246..218535|290 nt|1| | | |96 AA|96+16=112| |rev-compl|
EIVLTQSPTSMAVSQGERVT.ISCTASSSVS.....TSYLHWYQQKPGFPPRLLVYRT..
.....SSLASGVP.ARFSGSG..SGTSYTLTISSMEAEDAANYYCQQENSIP
>IMGT000063|IGKV3-17*01|Macaca mulatta_AG07107|F|V-REGION|1115668..1115954|287 nt|1| | | |95 AA|95+17=112| | |
EIVMTQSPATLSLSPGERAT.LSCRASQSV......SSSLAWYQQKPGQAPRLLIYDA..
.....SSRVTGIP.DRFSGSG..SGTDFTLTISSLEPEDVGVYFCQQESNWS
>AY963765|IGKV3-17*02|Macaca mulatta|F|V-REGION|206..492|287 nt|1| | | |95 AA|95+17=112| | |
EIVMTQSPATLSLSPGERAT.LSCRASQSV......SSRLAWYQQKPGQAPRLLIYDA..
.....SSRVTGIP.DRFSGSG..SGTDFTLTISSLEPEDVAVYFCQQESNWS
>AY963767|IGKV3-17*03|Macaca mulatta|F|V-REGION|206..492|287 nt|1| | | |95 AA|95+17=112| | |
EIVMTQSPATLSLSPGERAT.LSCRASQSV......SSRLAWYQQKPGQAPRLLIYDA..
.....SSRATGIP.DRFSGSG..SGTEFTLTISSLEPEDVAVYFCQQESNWS
>IMGT000063|IGKV3-24*01|Macaca mulatta_AG07107|F|V-REGION|1017311..1017597|287 nt|1| | | |95 AA|95+17=112| | |
EIVMTQSPATLSLSPGERAT.LSCRASQSV......SSSLAWYQQKPGQAPRLLIYGA..
.....SSRATGIP.DRFSGSG..SGTDFTLTISSLEPEDVAVYYCLQRSNWP
>NW_001099007|IGKV3-24*02|Macaca mulatta_17573|F|V-REGION|349905..350174|270 nt|1| | ||90 AA|90+17=107|partial in 3'|rev-compl|
EIVMTQSPATLSLSPGERAT.LSCRASQSV......SSSLAWYQQKPGQAPRLLIYGA..
.....SSRVTGIP.DRFSGSG..SGTDFTLTISSLEPEDVAVYYCQQ
>AY963763|IGKV3-24*03|Macaca mulatta|F|V-REGION|206..492|287 nt|1| | | |95 AA|95+17=112| | |
EIVMTQSPATLSLSPGERAT.LSCRASQSV......SSSLAWYQQKPGQAPRLLIYGA..
.....SSRATGIP.DRFSGSG..SGTEFTLTISSLEPEDVAVYYCLQRSNWP
>AY963764|IGKV3-24*04|Macaca mulatta|F|V-REGION|206..492|287 nt|1| | | |95 AA|95+17=112| | |
EIVMTQSPATLSLSPGERAT.LSCRASQSV......GSSLAWYQQKPGQAPRLLIYGA..
.....SSRATGIP.DRFSGSG..SGTDFTLTISSLEPEDVAVYYCLQRSNWP
>NW_001099007|IGKV3-26-1*01|Macaca mulatta_17573|ORF|V-REGION|411641..411927|287 nt|1| | | |95 AA|95+17=112| |rev-compl|
EIVMTQSPATLSLSPGERAT.LSCRASQSV......SSSLAWYQQKPGQAPKLLIYGA..
.....SSRATGIP.DGFSGSG..SGTEFTLTISSLEPEDVGVYYCQQDYSWP
>IMGT000063|IGKV3-31*01|Macaca mulatta_AG07107|F|V-REGION|900295..900581|287 nt|1| | | |95 AA|95+17=112| | |
EIVMTQSPATLSLSPGETAT.ISCRTSQSV......SSYLAWYQQKPGQAPRLLIYGA..
.....SSRATGIP.DRFSGSG..SGTDFTLTISSLEPEYFAVYYCQETSNLS
>NW_001099007|IGKV3-31*02|Macaca mulatta_17573|F|V-REGION|484506..484792|287 nt|1| | | |95 AA|95+17=112| |rev-compl|
EIVMTQSPATLSLSPGETAT.ISCRTSQSV......SSYLAWYQQKPGQAPRLLIYGA..
.....SSRATGIP.DRFSGSG..SGTDFTLTISSLEPEDFAVYYCQETSNLS
>IMGT000063|IGKV3-35*01|Macaca mulatta_AG07107|F|V-REGION|822457..822743|287 nt|1| | | |95 AA|95+17=112| | |
EIVMTQSPATLSLSPGERAT.LSCRASQSV......SSNLAWYQQKPGQAPRLLIYDA..
.....SNRATGIP.DRFSGSG..SGTDFTLTISSLEPEDVGVYYCQQESNWP
>AY963769|IGKV3-35*02|Macaca mulatta|F|V-REGION|206..492|287 nt|1| | | |95 AA|95+17=112| | |
EIVMTQSPATLSLSPRERAT.LSCRASQSV......SSNLAWYQQKPGQAPRLLIYYA..
.....SNRATGIP.DRFSGSG..SGTDFTLTISSLEPEDVGVYYCQQESNWP
>AY963771|IGKV3-35*03|Macaca mulatta|F|V-REGION|206..492|287 nt|1| | | |95 AA|95+17=112| | |
EIVMTQSPATLSLSPRERAT.LSCRASQSV......SSNLAWYQQKPGQAPRLLIYYA..
.....SNRATGIP.DRFSGSG..SGTDFTLTISSLEPEDVGVYYCQQESNWS
>IMGT000063|IGKV3-40*01|Macaca mulatta_AG07107|F|V-REGION|743811..744097|287 nt|1| | | |95 AA|95+17=112| | |
EIVMTQSPATLSLSPGETAT.LSCRASESV......GSYLAWYQQKPGQAPKLLVRSA..
.....YFRATGIP.DRFSGSG..SRTDFTLTISSLEPEDVGVYHCQQYNDLL
>NW_001099007|IGKV3-40*02|Macaca mulatta_17573|F|V-REGION|649776..650062|287 nt|1| | | |95 AA|95+17=112| |rev-compl|
EIVMTQSPATLSLSPGETAT.LSCRASESV......GSYLAWYQQKPGQAPKLLVHSA..
.....YFRATGIP.DRFSGSG..SRTEFTLTISSLEPEDVGVYHCQQYNDLL
>AY963762|IGKV3-40*03|Macaca mulatta|F|V-REGION|205..491|287 nt|1| | | |95 AA|95+17=112| | |
EIVMTQSPATLSLSPGETAT.LSCRASESV......GSYLAWYQQKPGQAPKLLVHSA..
.....YFRATGIP.DRFSGSG..SRTEFTLTISSLEPEDVGVYHCQQYNDLL
>IMGT000063|IGKV3-42*01|Macaca mulatta_AG07107|F|V-REGION|712347..712633|287 nt|1| | | |95 AA|95+17=112| | |
EIVMTQSPATLSLSPGERAT.LSCRASQSV......SSSLAWYQQKPGQAPRLLIYGA..
.....SSRATGIP.DRFSGSG..SGTEFTLTISSLEPEDVAVYYCQQNSNWP
>NW_001099007|IGKV3-42*02|Macaca mulatta_17573|F|V-REGION|685165..685451|287 nt|1| | | |95 AA|95+17=112| |rev-compl|
EIVMTQSPATLSLSPGERAT.LSCRASQSV......SSSLAWYQQKPGQAPRLLIYGA..
.....SSRATGIP.DRFSGSG..SGTEFTLTISSLEPEDFAVYYCQQYSNWP
>AY963768|IGKV3-42*03|Macaca mulatta|F|V-REGION|206..492|287 nt|1| | | |95 AA|95+17=112| | |
EIVMTQSPATLSLSPGERAT.LSCRASQSV......SSSLAWYQQKPGQAPRLLIYGA..
.....SSRATGIP.DRFSGSG..SGTEFTLTISSLEPEDFAVYYCQQYSNWP
>IMGT000063|IGKV3-53*01|Macaca mulatta_AG07107|F|V-REGION|576526..576812|287 nt|1| | | |95 AA|95+17=112| | |
QVILTQSPATLSLSPGERAT.LSCRASQSV......SSSLAWYQQKPGQAPRLLIYGA..
.....SSRATGIP.DRFSGSG..SGTEFTLTISSLEPEDFAVYYCQKYSSSP
>NW_001099007|IGKV3-53*02|Macaca mulatta_17573|F|V-REGION|822270..822556|287 nt|1| | | |95 AA|95+17=112| |rev-compl|
QVILTQSPATLSLSPGERAT.LSCRASQSV......SSSLAWYQQKPGQAPRLLIYGA..
.....SSRATGIP.DRFSGSG..SGTEFTLTISSLEPEDFAVYYCQKYSSSP
>IMGT000063|IGKV3-79*01|Macaca mulatta_AG07107|P|V-REGION|285340..285615|276 nt|1| | | |92 AA|92+17=109| | |
EVMLMQSTDSLSSSSGNRDF.LMCVCSQNT......GNSLAWYQQKPGKDPWFLIYGG..
.....YNRATTIP.EGFSGQG..SETDFSVTISCLEPEDFALLCLF*SL
>IMGT000063|IGKV3-88*01|Macaca mulatta_AG07107|P|V-REGION|237217..237492|276 nt|1| | | |92 AA|92+17=109| | |
DVTLMRSPDSLSLSSG*RDP.LVCGCHQNT......GSSLALYQRKPGEALWLLIYGV..
.....SSRATGIP.EGFSGHG..SETDFTVTISCLEHEDFVFFDLCWFK
>IMGT000063|IGKV3-93*01|Macaca mulatta_AG07107|P|V-REGION|187527..187802|276 nt|1| | | |92 AA|92+17=109| | |
DVMLMQSPDSLSLSSG*RDP.LVCGCRQNI......GSSLALYQRKPGEAPWLLIYGA..
.....SSRATGIP.EGFSGHG..SETDFTVTINCLEPEDFALLCLF*SL
>IMGT000063|IGKV3-96*01|Macaca mulatta_AG07107|P|V-REGION|157927..158202|276 nt|1| | | |92 AA|92+17=109| | |
GVILMQPPGSLSLSSGQRDS.LLCMCSQNI......GSSLALYQQKLGKAPWLLIYGG..
.....SSRATSIL.EGFSGHG..SETDLTATISCLKPEDFSLLCLFDLC
>AY963772|IGKV3S11*01|Macaca mulatta|F|V-REGION|234..520|287 nt|1| | | |95 AA|95+17=112| | |
QVILTQSPATLSLSPGERAT.LSCRASQSV......GSNLAWYQQKPGQAPRLLIYGA..
.....SSRATGIP.DRFSGSG..SGTDFTLTISSLEPEDVAVYYCLQRSNWP
>AY963766|IGKV3S5*01|Macaca mulatta|F|V-REGION|206..492|287 nt|1| | ||95 AA|95+17=112| | |
EIVMTQSPATLSLSPGERAT.LSCRASQSV......SSRLAWYKQKPGQAPRLLIYDA..
.....SSRATGIP.DRFSGSG..SGTEFTLTISSLEPEDVAVYLCQQESNWS
>AY963770|IGKV3S9*01|Macaca mulatta|F|V-REGION|206..492|287 nt|1| | | |95 AA|95+17=112| | |
EIVMTQSPATLSLSPGERAT.LSCRASQSV......SSYVAWYQQKPGQAPRLLIYGA..
.....SSRATGIP.DRFSGSG..SGTEFTLTISSLEPEDFAVYYCQQYSNWP
>IMGT000063|IGKV4-1*01|Macaca mulatta_AG07107|F|V-REGION|1318585..1318889|305 nt|1| | | |101 AA|101+11=112| |rev-compl|
DIVMTQSPDSLAVSLGERVT.INCKSSQSLLYSSNNKNYLAWYQQKPGQAPKLLIYWA..
.....STRESGVP.NRFSGSG..SGTDFTLTISGLQAEDVAVYYCQQYYSTP
>NW_001099007|IGKV4-1*02|Macaca mulatta_17573|F|V-REGION|93796..94100|305 nt|1| | | |101 AA|101+11=112| | |
DIVMTQSPDSLAVSLGERVT.INCKSSQSLLYSSNNKNYLAWYQQKPGQAPKLLIYWA..
.....STRESGVP.NRFSGSG..SGSDFTLTISGLQAEDVAVYYCQQYYSTP
>IMGT000063|IGKV4-8*01|Macaca mulatta_AG07107|P|V-REGION|1221364..1221668|305 nt|1| | | |101 AA|101+11=112| | |
DIRLTQSPGLRAVSPGEMVI.INCKSSQSLLNTFDNKNYFAWH*QKPGQAPKMLISWA..
.....STQAPGVP.DQFIGSR..SETDYTLTFSSLQAGDVAVYYCQHYYSTS
>NW_001099007|IGKV4-8*02|Macaca mulatta_17573|P|V-REGION|191490..191794|305 nt|1| | | |101 AA|101+11=112| |rev-compl|
DIRLTQSPGLLAVSPGEMVI.INCKSSQSLLNTFDNKNYFAWH*QKPGQAPKMLISWA..
.....STQAPGVP.DQFIGSR..SETDYTLTFSSLQAGDVAVYYCQHYYSTS
>X79831|IGKV4S1*01|Macaca mulatta|ORF|V-REGION|34..338|305 nt|1| | | |101 AA|101+11=112| | |
DIVMTQSPDSLAVSLGERVT.INCKSSRSLLYSSNNKNYLAWYQQKPGQAPKLLVYWA..
.....STRESGVP.NRFSGSG..SGTDFTLTISGLQAEDVAVYYCQQYYSSP
>IMGT000063|IGKV5-11*01|Macaca mulatta_AG07107|F|V-REGION|1201994..1202280|287 nt|1| | | |95 AA|95+17=112| |rev-compl|
ETILTQSAAFMSATPGDKVT.ISCKASQDI......DDDMNWYQLEPGEAPKLIIKDA..
.....TTLVSGIP.PRFSGSG..YGTDFTLTINNVESEDAAYYFCLQHDNFP
>NW_001099007|IGKV5-11*02|Macaca mulatta_17573|F|V-REGION|210859..211145|287 nt|1| | | |95 AA|95+17=112| | |
ETILTQSAAFVSATPGDKVT.ISCRAGQDI......DDDMNWYQQEPGEAPKLIIKDA..
.....TTLVSGIP.PRFSGSG..YGTDFTLTINNVESEDAAYYFCLQHDNFP
>IMGT000063|IGKV5-5*01|Macaca mulatta_AG07107|F|V-REGION|1269201..1269487|287 nt|1| | | |95 AA|95+17=112| |rev-compl|
EMILTQSPAFVSATPGDKVT.ISCRAGQDI......DDDMNWYQQEPGEAPKLIIKDA..
.....TTLVSGIP.PQFSGSG..YGIDFTLTINSMKSEDTAYYFCQQRDNTP
>IMGT000063|IGKV6-47*01|Macaca mulatta_AG07107|F|V-REGION|645834..646118|285 nt|1| | | |95 AA|95+17=112| | |
DIVMTQSPAFVSVTPGEKVT.ITCQVSEGI......SNYLHWYQQKPDQAPKLFIQYA..
.....SQSISGVP.SRFTGSG..SGTDFTFTISSLEVEDAATYYCQQGNKHP
>IMGT000063|IGKV6-55*01|Macaca mulatta_AG07107|F|V-REGION|547341..547627|287 nt|1| | | |95 AA|95+17=112| | |
EIVLTQSPAFRSVTLKEKVT.ITCQASQSI......GSSLHWYQQKPDQSPKLLIKYA..
.....SQSISGVP.SRFSGSG..SGTDFTLTINSLEAEDAATYYCQQSSSFP
>IMGT000063|IGKV6-67*01|Macaca mulatta_AG07107|P|V-REGION|406620..406906|287 nt|1| | | |95 AA|95+17=112| | |
EVVLT*APTFLSVTPGETVT.ITCWVSQGI......GNYLHWHQ*KEDQAPKLLTNYA..
.....SQSILGCS.WRFSGSG..SATDLACYSYKHVFPCPVGLFSTIKVVKY
>IMGT000063|IGKV7-13*01|Macaca mulatta_AG07107|P|V-REGION|1176743..1177041|299 nt|1| | | |99 AA|99+13=112| | |
DIVLTQSPASLAVSPGQRAT.ITCRASESVSV..FGINLIHWYQQKPGQPPKLLIYQA..
.....SNKDTGVP.ARFSGSG..SGTDFTLTINPVEADDAADYYCLQSKNSP
>NW_001099007|IGKV7-13*02|Macaca mulatta_17573|F|V-REGION|236056..236354|299 nt|1| | | |99 AA|99+13=112| |rev-compl|
DIVLTQSPASLAVSPGQRAT.ITCRASETVSF..FGINLIHWYQQKPGQPPKLLIYQA..
.....SNKDTGVP.ARFSGSG..SGTDFTLTINPVEADDAADYYCLQSKNSP

</pre>
<hr />





<a href="/genedb/doc">-> IMGT/GENE-DB Documentation</a><br />
<a href="/genedb/">-> IMGT/GENE-DB Query page</a><br />
<a href="/genedb/directlinks">-> IMGT/GENE-DB direct links</a><br />
<a href="/genedb/stats">-> IMGT/GENE-DB statistics</a><br />
<a href="/IMGTgenedbdoc/programversions.html">-> IMGT/GENE-DB program versions</a><br />
<a href="/IMGTgenedbdoc/dataupdates.html">-> IMGT/GENE-DB data updates</a>

<hr />

<p class="menu">
<a href="/">IMGT Home page</a>
| <a href="/IMGTrepertoire/">IMGT Repertoire (IG and TR)</a>
| <a href="/IMGTrepertoireMH/">IMGT Repertoire (MH)</a>
| <a href="/IMGTrepertoireRPI/">IMGT Repertoire (RPI)</a>
| <a href="/IMGTindex/">IMGT Index</a>
| <a href="/IMGTScientificChart/">IMGT Scientific chart</a>
| <a href="/IMGTeducation/">IMGT Education</a>
</p>



<span class="logosFoot">
<a href="https://www.cnrs.fr"><img src="/images/cnrs.png" alt="logo CNRS" style="vertical-align:middle;" /></a>
<a href="https://www.umontpellier.fr/"><img src="/images/logos/universite_mtp.png" alt="Universit&#233; de Montpellier" style="width:60px;vertical-align:middle;" /></a>
<a href="https://ec.europa.eu/info/research-and-innovation_en"><img src="/images/UE.png" alt="logo UE" style="vertical-align:middle;" /></a>
</span>



<p>&#169;  Copyright 1995-2021 IMGT&reg;, the international ImMunoGeneTics information system&reg;
			| <a href="/about/termsofuse.php">Terms of use</a> | <a href="/about/aboutus.php">About us</a> | <a href="mailto:contact-imgt@igh.cnrs.fr">Contact us</a> | <a href="/about/CitingIMGT.php">Citing IMGT</a></p>

<!-- #echo var=$LAST_MODIFIED -->

</body>
</html>

</body>
</html>